### PR TITLE
chemistry "rebalance" 

### PR DESCRIPTION
**Pull request by BarteG44**
## About The Pull Request
Moves the chem dispenser higher up the tech tree to coax people into doing ghetto chem and to additionally help with that a new chemistry cargo category is added to the outpost which contains full chemical jugs of a few chemicals plus the starter kit crate from the medical category and a tool crate. The category uses a new (soulful) chemical crate.

Also adds the previously mentioned chemical jugs which are 150u plastic containers that cannot be stored in a backpack and have inconvenient transfer amounts to "balance" how cheap they are. They also come with a little fluff feature in the form of a tamper seal which lets you know that no one touched anything inside the jug before you

image
image
image 
image

## Why It's Good For The Game
Hopefully we'll see less chem dispensers around

## Changelog

:cl:
add: Added chemical crates
add: Added chemical jugs
balance: moved the chem dispenser to applied bluespace research
/:cl:

### CODE REVIEW

**Comment by Zevotech** [comment]
i actually like this. good job!

**Comment by zestybastard** [comment]
Hi, chem player here. I would like to discuss a few balance (and pricing) concerns with regards to this PR, some of which you may not have taken into consideration so I would like to raise them to your attention. They are as follows:

- Right now one of the balancing factors for these 150u jugs is their inability to transfer "convenient" amounts of a reagent, however this can be subverted using syringes. Consider making them incompatible with syringes to necessitate the use of transfer beakers.
- A 150u jug can make a sizable (0,3,7,0) explosion when used as a handheld gunpowder grenade in conjunction with the capping mechanic and a lighter.
- If you use compounds like RDX, TaTP, or Nitroglycerin, you can generate up to (0,4,8,0) (TaTP) and even (1,6,12,0) (Nitroglycerin) chain reaction containers. This and the above point makes them _very_ capable weapons for a 500 credit price tag.
- When uncapped, these jugs can also be thrown to splash 150u of a reagent (CLF3, Nitric Acid, etc.) at a time, making them extremely lethal throwing weapons.
- The above points are balanced by their bulky status however, so I am fine as long as you've considered the potential of these things.

**Comment by BarteG44** [comment]
>Right now one of the balancing factors for these 150u jugs is their inability to transfer "convenient" amounts of a reagent, however this can be subverted using syringes. Consider making them incompatible with syringes to necessitate the use of transfer beakers.

this is a good point

>A 150u jug can make a sizable (0,3,7,0) explosion when used as a handheld gunpowder grenade in conjunction with the capping mechanic and a lighter.
If you use compounds like RDX, TaTP, or Nitroglycerin, you can generate up to (0,4,8,0) (TaTP) and even (1,6,12,0) (Nitroglycerin) chain reaction containers. This and the above point makes them very capable weapons for a 500 credit price tag.
When uncapped, these jugs can also be thrown to splash 150u of a reagent (CLF3, Nitric Acid, etc.) at a time, making them extremely lethal throwing weapons.

i don't think these are such a big problem considering that the jugs have a reduced throwing range and you're limited to holding 2 at the same time at max. Also metamaterial beakers aren't _that_ hard to get, not to mention that PvP isn't really a common thing here

**Comment by Erikafox** [comment]
> i don't think these are such a big problem considering that the jugs have a reduced throwing range and you're limited to holding 2 at the same time at max. Also metamaterial beakers aren't _that_ hard to get, not to mention that PvP isn't really a common thing here

You could hack it together so they only splash so much at once. I think it'd make sense with the limited pouring.

**Comment by BarteG44** [comment]
>You could hack it together so they only splash so much at once. I think it'd make sense with the limited pouring

I'll sure give it a try

**Review by thgvr — APPROVED** [review]
These sprites are fine for now, and I like the idea

File: code/modules/reagents/reagent_containers/jug.dm [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+			return
+		. += "<span class='green'>The tamper seal is <b>intact</b>.</span>"
+	else
+		. += "<span class='warning'>The tamper seal is <b>broken</b>.</span>"
+
+/obj/item/reagent_containers/glass/chem_jug/open
+	cap_on = FALSE
+
+/obj/item/reagent_containers/glass/chem_jug/carbon
+	name = "jug of carbon"
```
Comment (thgvr) [inline code/modules/reagents/reagent_containers/jug.dm]: Could you name these all a little better? something like `jug (carbon)`?

Comment (BarteG44) [reply code/modules/reagents/reagent_containers/jug.dm]: Personally i wouldn't call that better but alright. I'm not a big fan of adding stuff in parentheses to names

File: code/modules/reagents/reagent_containers/jug.dm [hunk 2]
```diff
@@ -40,6 +40,43 @@
...
+						"<span class='userdanger'>[M] is splashed with something!</span>")
+		for(var/datum/reagent/A in reagents.reagent_list)
+			R += "[A.type]  ([num2text(A.volume)]),"
+
+		if(thrownby)
+			log_combat(thrownby, M, "splashed", R)
+		reagents.expose(target, TOUCH, 0.3)
+
+	else if(bartender_check(target) && thrown)
+		visible_message("<span class='notice'>[src] lands onto the [target.name] without spilling a single drop.</span>")
```
Comment (Erikafox) [inline code/modules/reagents/reagent_containers/jug.dm:64]: pass me a jug of ethanol, mr bartender

Comment (BarteG44) [reply code/modules/reagents/reagent_containers/jug.dm:64]: what ctrl+c ctrl+v does to a mf

**Review by MarkSuckerberg — APPROVED** [review]
looks good

**Comment by rye-rice** [comment]
you look good

**Comment by MarkSuckerberg** [comment]
:flushed:

also I just realised I didn't hit merge so thanks for making me come back

**Pull request merged**